lib/Listener/FileListener.php

```diff
@@ -114,53 +113,44 @@ public function handle(Event $event): void {
 					return;
 				}
 				$files = $this->storageService->getFilesInMount($mount->getNumericStorageId(), $node->getId(), 0, 0);
+				$fileRefs = [];
+
 				foreach ($files as $fileId) {
 					$node = current($this->rootFolder->getById($fileId));
 					if (!$node instanceof File) {
 						continue;
 					}
-					try {
-						$fileHandle = $node->fopen('r');
-					} catch (LockedException|NotPermittedException $e) {
-						$this->logger->error('Could not open file ' . $node->getPath() . ' for reading', ['exception' => $e]);
-						return;
-					}
-					foreach ($userIds as $userId) {
-						try {
-							$source = new Source($userId, 'file: ' . $node->getId(), $fileHandle, $node->getMtime(), $node->getMimeType());
-						} catch (InvalidPathException|NotFoundException $e) {
-							$this->logger->error('Could not find file ' . $node->getPath(), ['exception' => $e]);
-							break;
-						}
-						$this->langRopeService->deleteSources($userId, [$source]);
+
+					if (!$this->allowedMimeType($node)) {
```

Comment:
getFilesInMount should already select the right files, now that you made it use the right mime type list

Reply:
ah, didn't see it was fetching files from there, thanks